docker_test.go

```diff
@@ -342,3 +343,21 @@ func runKaniko(t *testing.T, imageToPush, kanikoImage string) string {
 	assert.NoError(t, gofrogcmd.RunCmd(imageRunnder))
 	return filepath.Join(workspace, imageNameWithDigestFile)
 }
+
+func TestXrayDockerScan(t *testing.T) {
+	initXrayCli()
+	validateXrayVersion(t, scan.DockerScanMinXrayVersion)
+
+	// Pull alpine image from docker repo
+	imageTag := path.Join(*tests.DockerRepoDomain, tests.DockerScanTestImage)
+	dockerPullCommand := corecontainer.NewPullCommand(container.DockerClient)
+	dockerPullCommand.SetImageTag(imageTag).SetRepo(*tests.DockerVirtualRepo).SetServerDetails(serverDetails).SetBuildConfiguration(new(utils.BuildConfiguration))
+	assert.NoError(t, dockerPullCommand.Run())
+
+	// Run docker scan on alpine image
+	output := runAuditCmdWithOutput(t, container.DockerClient.String(), "scan", tests.DockerScanTestImage)
```

Comment:
Since the ```runAuditCmdWithOutput``` API doesn't do anything that is specific to "audit" or "xray", I suggest renaming it to ```ExecWithOutput``` and moving it to ```tests/utils.go``` right next to the ```func (cli *JfrogCli) Exec(args ...string) error``` function.

We should probably modify it a bit, and make its signature look more like the ```Exec``` for consistency sake.